<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatdig</class>
 <widget class="QWidget" name="chatdig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="tB_rs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_inword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>721</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_fiie">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件传输</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_emo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>表情</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_vedio">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>视频聊天</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_send">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发    送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clear">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清    空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
